### DISCUSSION TITLE
.to_graphql_query for Mutations and Queries?

### DISCUSSION THREAD

**Discussion by chughes87** [rmosolgo/graphql-ruby#4950] [Ideas]
I would like to be able to get a template query or mutation that I would be able to use with a given Query or Mutation implementation. It seems fairly possible with a little introspection? I always struggle to get every part of the query/mutation syntax right when implementing my frontend. This little utility could help expedite the process.

**Comment by rmosolgo** [comment]
Hey, thanks for the suggestion! I don't have plans to work on it, but if I was looking for a quick way to write up queries, I would use GraphiQL. It has a great autocomplete feature and realtime validation, so it will tell you if anything is wrong with the query. Do you have that set up for development?
